client/source/class/cv/io/transport/LongPolling.js

```diff
@@ -247,7 +317,16 @@ qx.Class.define('cv.io.transport.LongPolling', {
               readyState = 'COMPLETED';
               break;
           }
-          this.error('Error! Type: "' + str + '" ExceptionObject: "'+ excptObj + '" readyState: ' + readyState);
+
+          this.error(
+            'Error! Type: "' +
+              str +
+              '" ExceptionObject: "' +
+              excptObj +
+              '" readyState: ' +
+              readyState
+          );
+
```

Comment:
Would look better by using the `-Syntax and stay in one line, i.e.
```
  this.error(`Error! Type: "${str}" ExceptionObject ...`);
```

Reply:
I will not change every concatenated string to use backticks, thats not the purpose of this pull request. Please note that it still would add linebreaks if the line exceeds the allowed width, even with backticks. All I can do is to increase the allowed line width, to avoid some of the line breaks.